src/gt4py/next/program_processors/runners/dace_iterator/workflow.py

```diff
@@ -115,15 +119,27 @@ class Meta:
         model = DaCeTranslator
 
 
+class CompiledDaceProgram(stages.CompiledProgram):
+    sdfg_program: dace.CompiledSDFG
+    sdfg_scalar_args: dict[str, int]  # map arg to its position in program ABI
```

Comment:
I am not sure why all scalar arguments should be integers?
For example the timestep size is a scalar but not an integer (usually).

Reply:
This integer value represents the position of a scalar arguments in the ABI call signature.